README.md

```diff
@@ -9,6 +9,7 @@ Some rails specific options.
 
 ```ruby
 set :rails_env, 'staging'                  # If the environment differs from the stage name
+set :rails_assets_groups, nil              # RAILS_GROUPS env value for the assets:precompile task
```

Comment:
Not sure. I think it could be confusing, so someone can think this envvar is applied to ALL tasks. The main point is to add this RAILS_ENV only for `assets:precompile`.

If you think it's ok, i'll change. I don't think a lot of people use `RAILS_GROUPS`.